data/unmatched_slugs.yml

```
@@ -0,0 +1,33 @@
+  "city-of-aberdeen": "aberdeen"
```

Could you outdent all these, for tidiness?

```suggestion
"city-of-aberdeen": "aberdeen"
```